styles/_utilities.css

```diff
@@ -0,0 +1,42 @@
+.buddies {
+  font-weight: 600;
+  color: var(--color-lightblue);
+}
+
+.journal-heading {
+  line-height: 1.333;
+}
+
+.comprehension {
+  letter-spacing: -0.26px;
+}
```

Comment:
Utility classes should be more descriptive so that their function can be understood quickly, such as margin-t10 (or mt-10). And i think the marked classes are better kept in the components? Or is there a reason why these are included in the utility classes that i don't yet understand?